src-ghc/Pact/GasModel.hs

```diff
@@ -130,29 +138,55 @@ writeCSV results = do
   BSL8.writeFile "gas-model.csv" (header <> content)
 
 
+data Option = Option
+  { _oFilter :: Maybe T.Text
```

Comment:
Is it possible for the parser to read in a list of Text instead? Or refactor the code that handles splitting along "`|`" out of main and into its own function.